components/navbar/navbar.css

```diff
@@ -0,0 +1,127 @@
+.navbar-container {
+    display: flex;
+    align-items: center;
+    background-color: var(--bg-color-1);
+    box-shadow: 0px 10px 6px -6px var(--primary-color);
+    width: 100%;
+    padding: 0.5rem;
+    top: 0;
+}
+.navbar-logo-container {
+    width: 30%;
+}
```

Comment:
add 1 line space after each css block for better readability

Reply:
ok i will do it